docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,337 @@
+
+        or::
+
+         [SRU][B/F/J][PATCH 0/1] UBUNTU: [Config]: Add support for modifying LDT
+
+      * If the patch has to be applied to a specific derivative for multiple
+        release this is the recommended format::
+
+         [SRU][B:linux-kvm,F:linux-kvm] [PATCH 0/1] UBUNTU: [Config] kvm: Add support for modifying LDT
+
+#. If the patch requested doesn't come from upstream it must contain one of
+   the following on the Subject line after the release name string
+   described previously:
+
+   .. list-table:: Example table
+      :header-rows: 1
+
+      * - Descriptor
+        - Meaning
+      * - UBUNTU: [Packaging]
+        - This is a patch to the Ubuntu Packaging, including the contents
+          of the various debian* directories (where not covered by a more
+          specific categorisation)
+      * - UBUNTU: [Config]
+        - This is an update to the kernel configuration as recorded in the
+          debian.<branch>/config directory. Please check
+          ``debian.master/config/README.rst`` or `here
+          <https://discourse.ubuntu.com/t/kernel-configuration-in-ubuntu/35857>`__
+          for more information about the config format.
+      * - UBUNTU: ubuntu
+        - This is an update to an Ubuntu specific driver in the ubuntu/
+          directory.
+      * - UBUNTU: SAUCE:
+        - This is a patch to the kernel code that has not been applied on
+          mainline (Linus' tree). This category covers the following cases:
+
+             #. The submitter has either authored the patch or obtained the
+                patch from a non-upstream source.
+             #. The patch has been applied to an upstream tree but not yet
+                merged on mainline.
+             #. The patch is never expected to be submitted upstream but is of
+                enough value for Ubuntu to carry it in our tree.
+             #. The patch has been submitted to upstream but is of enough
+                value for Ubuntu to carry it in our tree regardless of
+                upstream acceptance.
+
+      * - UBUNTU: [Debian]
+        - This patch changes the debian rules files (``debian/rules,
+          debian*/rules.d/*`` and possibly ``debian*/control`` and
+          ``debian*/control.d/*``).
+
+   Example::
+
+     [SRU][FOCAL][PATCH 2/2] UBUNTU: SAUCE: shiftfs: prevent ESTALE for LOOKUP_JUMP lookups
+
+Comment Body
+------------
+
+#. Every patch associated with a launchpad bug must have a link to the bug
+   in the commit's comment section in the form of a "BugLink" block. A
+   "BugLink" block must immediately follow the subject line and be the
+   first text in the body of the commit comment. A "BugLink" block consists
+   of:
+
+   #. A blank line.
+   #. One or more lines containing "BugLink:" and a url to the launchpad
+      bug. That url must be of the format:
+      "https://bugs.launchpad.net/bugs/<bug-id>" where <bug-id> is the
```

Comment:
This is a minor thing.
but it's rendered as a link.
Escaping any of the characters would help.

```suggestion
      "https\://bugs.launchpad.net/bugs/<bug-id>" where <bug-id> is the
```

Reply:
Done. Thanks!